### ISSUE TITLE
*load_save* hides Purchase Popup Dialog

### ISSUE THREAD

**Issue by JoshMayberry** [SublimeLinter/SublimeLinter#1749]
If the setting `lint_mode` is set to `load_save`, the "Thank you for trying Sublime Text, would you like to purchase it" dialog shows up behind the window instead of appearing in front of it.

I think this happens because the linter interacts with the window after the save, which shifts the focus away from the popup and back to the window.

If the popup box is left open, it causes Sublime Text to react strangely.

**Comment by kaste** [comment]
> I think this happens because the linter interacts with the window after the save, 

Do you have something specific in mind here?

**Comment by braver** [comment]
It's inevitable that packages interact with the window after a save. If there is a way to solve it that lies with Sublime Text itself. The nag window is not something packages should have to worry about IMO.

**Comment by JoshMayberry** [comment]
> Do you have something specific in mind here?

Would it be possible to detect if the popup has appeared?
If it has, the program could run the linter yet. 
It could then either wait for the popup to close or for the user to save it a second time.
This behavior could be configured by the user as a setting variable.

**Comment by braver** [comment]
Well, yeah, no. We're a plugin. We're not going to worry about the nag screen on that level. Also, we can't, again because we're a plugin. Better report this to Sublime Text itself. Or buy the license ;)

**Comment by kaste** [comment]
@JoshMayberry I could not reproduce it.  I removed my license, and then it took long enough to get at 3 NAG dialogs, but they were all fine at the top. This is using Windows. Actually, there is not much difference between "load_save" mode or background mode, we almost do the same stuff on post save, so I wouldn't expect a difference here. 

TL;DR I would think this is probably coming from a different plugin. Maybe you have an auto-formatter on save installed?  

@braver a close *and* wontfix? ... I know you well enough, you did not reproduce it, did you?

**Comment by braver** [comment]
You know me too well 😉

The nag popup is another window. Wether or not activity in the main window moves the main window to the front is probably OS specific. We’re not actively stealing focus or anything like that. We’re also not the only package that might draw highlights or otherwise update something in the main window at certain events or even with certain intervals. It’s entirely reasonable for packages to do so.

So it might be a real problem, but that’s not the point. It’s not _our_ problem.

**Comment by braver** [comment]
If you want to have the issue looked into further, report it here: https://github.com/sublimehq/sublime_text/issues
